omod/src/main/webapp/pages/metadata/privileges/privilege.gsp

```diff
@@ -71,8 +71,8 @@
             initialValue : ui.encodeHtmlContent(privilege.privilege)
         ])}
     <% } else{ %>
-        <b>${ui.message("general.name")}:</b> ${privilege.privilege}
-        <input type="hidden" name="privilegeName" value="${privilege.privilege}" />
+        <b>${ui.message("general.name")}:</b> ${ui.encodeHtmlContent(privilege.privilege)}
```

Comment:
hi @isears  does this bring much difference than using <c:out value='${privilege.privileget}'/>

Reply:
Good question Herbert, I can do either.

Reply:
I don't think you can use `<c:out>` tags outside of `.jsp` files